### PR TITLE
Add 30 second timeout to requests

### PR DESCRIPTION
**Pull request by steveoh**
## Description of Changes
This pull requests adds a timeout to requests

### Test results and coverage

```
Name                 Stmts   Miss     Cover   Missing
-----------------------------------------------------
forklift\arcgis.py      94     16    82.98%   34-44, 71, 125-128, 137-139
forklift\cli.py        264     51    80.68%   88, 125-126, 206-211, 244, 349-360, 364-373, 383-424
forklift\core.py       182      2    98.90%   119, 324
forklift\lift.py       166     24    85.54%   179-199, 209-211, 223-225, 240, 248-250, 284, 299
forklift\models.py     191      8    95.81%   99, 294-297, 353, 397-398
-----------------------------------------------------
TOTAL                  997    101    89.87%

5 files skipped due to complete coverage.
-----------------------------------------------------------------------------
161 tests run in 353.896 seconds.
10 skipped (151 tests passed)
```

### Speed test results

```
Speed Test Results
Dry Run: 4.15 minutes
Repeat: 3.1 minutes
```

### CODE REVIEW

**Comment by steveoh** [comment]
this is ready meow.

File: src/forklift/arcgis.py [hunk 1]
```diff
@@ -114,12 +114,18 @@ def _fetch(self, url):
...
-
-        ok = self._return_false_for_status(r.json())
+        
+        try:
+            r = requests.post(url, data=data, timeout=30)
+            r.raise_for_status()
+
+            ok = self._return_false_for_status(r.json())
+        except requests.exceptions.Timeout:
+            return False
```
Comment (stdavis) [inline src/forklift/arcgis.py]: This would skip the sleep line below. Not sure how important that is but maybe it should be changed to `ok = False`.

Comment (steveoh) [reply src/forklift/arcgis.py]: Would that matter if it took 30 seconds to timeout?

Comment (stdavis) [reply src/forklift/arcgis.py]: Good point.

**Comment by stdavis** [comment]
LGTM

**Pull request merged**